### PR TITLE
net-irc/inspircd 3.4.0

### PR DESCRIPTION
**Pull request by clinew**

### CODE REVIEW

File: net-irc/inspircd/inspircd-3.4.0.ebuild [hunk 1]
```diff
@@ -0,0 +1,123 @@
+# Copyright 1999-2019 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI="7"
+
+inherit toolchain-funcs
+
+DESCRIPTION="Inspire IRCd - The Stable, High-Performance Modular IRCd"
```
Comment (juippis) [inline net-irc/inspircd/inspircd-3.4.0.ebuild]: I think you could remove the `Inspire IRCd - ` part, but don't mind either way.

File: net-irc/inspircd/inspircd-3.4.0.ebuild [hunk 2]
```diff
@@ -0,0 +1,123 @@
...
+	mysql? ( dev-db/mysql-connector-c:= )
+	pcre? ( dev-libs/libpcre )
+	postgres? ( dev-db/postgresql:= )
+	re2? ( dev-libs/re2:= )
+	sqlite? ( >=dev-db/sqlite-3.0 )
+	ssl? ( dev-libs/openssl:= )
+	tre? ( dev-libs/tre )"
+DEPEND="${RDEPEND}"
+
+DOCS=( docs/. )
```
Comment (juippis) [inline net-irc/inspircd/inspircd-3.4.0.ebuild]: Hmm, does this really work? Legit question, I have a feeling I tried this once and it didn't work.

File: net-irc/inspircd/inspircd-3.4.0.ebuild [hunk 3]
```diff
@@ -0,0 +1,123 @@
...
+		--manual-dir="/usr/share/man")
+	CXX="$(tc-getCXX)" ./configure "${myconf[@]}"
+}
+
+src_compile() {
+	emake LDFLAGS="${LDFLAGS}" CXXFLAGS="${CXXFLAGS}" $(usex debug 'INSPIRCD_DEBUG=2' '') INSPIRCD_VERBOSE=1
+}
+
+src_install() {
+	emake DESTDIR="${D}" install
```
Comment (juippis) [inline net-irc/inspircd/inspircd-3.4.0.ebuild]: You can just call `default` here, which also calls `einstalldocs` for you.

File: net-irc/inspircd/inspircd-3.4.0.ebuild (hunk 1 above)
Comment (clinew) [reply net-irc/inspircd/inspircd-3.4.0.ebuild]: I'll refactor it a bit.

File: net-irc/inspircd/inspircd-3.4.0.ebuild (hunk 2 above)
Comment (clinew) [reply net-irc/inspircd/inspircd-3.4.0.ebuild]: Looks like it:

```
test /usr/portage/net-irc/inspircd # ls /usr/share/doc/inspircd-3.4.0/ 
Doxyfile.bz2  LICENSE.txt.bz2  conf  sql
test /usr/portage/net-irc/inspircd # ls /home/frostsnow/inspircd/docs/ 
Doxyfile  LICENSE.txt  conf  sql
```

File: net-irc/inspircd/inspircd-3.4.0.ebuild (hunk 3 above)
Comment (clinew) [reply net-irc/inspircd/inspircd-3.4.0.ebuild]: Okay, done.

**Comment by clinew** [comment]
The issue which I was discussing on Bugzilla (https://bugs.gentoo.org/show_bug.cgi?id=693692) is only kind of related to this one, so I didn't included it in a tag.  Updated version pushed!

File: net-irc/inspircd/inspircd-3.4.0.ebuild (hunk 2 above)
Comment (juippis) [reply net-irc/inspircd/inspircd-3.4.0.ebuild]: (I think I got it - FYI for anyone interested: I tried with `*`, not `.`)

**Comment by clinew** [comment]
Thanks for the review and merge!

**Pull request closed without merging**